cookbooks/wiki/templates/default/mw-ext-Wikibase.inc.php.erb

```diff
@@ -65,6 +65,7 @@ call_user_func( function() {
 // $wgWBClientSettings['showExternalRecentChanges'] = true;
 
 $wgWBClientSettings['namespaces'] = [ NS_MAIN ];
+$wgWBClientSettings['repoSiteName'] = "Data Items";
```

Comment:
The [`repoSiteName`](https://doc.wikimedia.org/Wikibase/master/php/md_docs_topics_options.html#autotoc_md357) variable should be set to a message key. That way, administrators can translate the site name into other languages on-wiki using MediaWiki: messages.